src/index.ts

```diff
@@ -92,33 +95,66 @@ interface Compiler {
 }
 
 /**
- * Match the compiler based on file name, extension name, media/type or by content sniffing.
+ * Is a string a file system path?
  *
- * Iterates through the compilers and returns that matches any of the criteria.
+ * Several functions in this module allow for content to be passed as a
+ * string of raw content (e.g. `This is some Markdown`) or as a file system
+ * path. This function, assesses whether a file is a file system path or not.
  *
- * @param filePath The file path
- * @param mediaType The media type (e.g. `text/plain`) or the corresponding extension (e.g. `txt`)
+ * If the string starts with `/`, `./` or `../` then it is assumed to be a path
+ * but the presence of the path is not checked. For all other strings, this function
+ * returns true if the file exists.
+ *
+ * @param content The string to assess.
+ */
+export function isPath(content: string): boolean {
+  if (
+    content.startsWith('/') ||
+    content.startsWith('./') ||
+    content.startsWith('../')
```

Comment:
Could you share the reason for not checking the presence of the provided path please?
Reason being that these cases don't seem to take into account Windows file system paths. if we don't mind actually accessing the file system we could use [`fs.lstat`](http://nodejs.org/api/fs.html#fs_fs_lstat_path_options_callback), otherwise we might want to add `\` as an another separator.

Reply:
The reason for not checking presence here is because "if the content looks like a path then the user probably meant it to be a path". That is, if I try to convert the string "./file.md" then, if that file doesn't exist, I want the Markdown converter to give me an error message. I don't want the `match` function to try and find some other converter that acts on a plain strings. We use `fs.existsSync` in this function just to determine if a string that may or may not be a file, is actually a file. Does that make sense?

So for now I think I'd prefer to add \ as an another separator. Maybe make it a regex?

Reply:
Got it. Both the explanation and proposed solution makes sense. Thanks

Reply:
Changes made accordingly. Thanks!